docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -65,16 +65,6 @@ import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
 
 {{"demo": "LocalizationLuxon.js"}}
 
-:::warning
```

Comment:
Fixed by https://github.com/moment/luxon/issues/1197

We might not cover all use case (see the linked issue for context) but we did not have any negative feedback since the issue have been merged on Luxon.

Maybe we can keep a warning but less scaring.
Something like "The support for Luxon macro-token (`D`, `DD`, `T`, `TT`, ...) might be imperfect, if you encounter any weird behavior, please feel free to open an issue to help us improve it".

Reply:
I'm honestly fine with removing it. If it turns our to still be problematic, we can reintroduce a less scary callout instead. 👌